gulpfile.js

```diff
@@ -24,16 +24,21 @@ var argv = yargs
   .command('build', 'generate optimimized third party library for production',
     function(yargs) {
       argv = yargs
-        .usage('Usage: $0 build [--minify]')
+        .usage('Usage: $0 build [--minify] [--output_directory]')
         .option('minify', {
           describe: 'Whether to minify third-party dependencies'
-        }).argv;
+        })
+        .option('output_directory', {
+          describe: 'A path to the directory where the files will be generated'
+        })
+        .argv;
```

Comment:
It was on the same line as the ``)`` before, though. It's also not a method, so it doesn't quite flow as well as the other one.